ondine/homework2.md

```diff
@@ -0,0 +1,246 @@
+### Ondine Gallatin
+
+### Homework #2
+
+Due Monday, 1/19
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** _(Easy, 10% of total time)_
+
+**a)** Write a function `fractionString(n,d)` which takes 2 parameters (n,d) and returns a string.  For example, 
+`fractionString(7,4)` should return "1 3/4", and `fractionString(3,3)` should (for now) return "1 0/3".  As before, assume that _n_ and _d_ are both positive integers.
+
+```
+function fractionString(n, d) {
+  var numeratorTwo = n % d;
+  var number = Math.floor(n / d);
+  var result = number + ' ' + numeratorTwo + '/' + d;
+  return result;
+}
+```
```

Comment:
Great Job. This is technically correct, but I would advise against naming a variable "number". They are all numbers so that variable name does not contribute to the overall legibility of the code. something like "wholeNumber" or  "whlNum" might be better.